.markdownlint.json

```diff
@@ -29,8 +29,7 @@
       "code", "pre",
       "i", "strong", "em", "b",
       "sup",
-      "anonymous",
-      "prot"
+      "anonymous"
```

Comment:
`anonymous` isn't a valid element. Where did you hit this? It should probably be escaped.

Reply:
I don't see any `<anonymous>` element in the generated HTML files so I'd say you can remove it completely from here. Well, apart from a comment in Japanese translation.

All of those comments should be removed BTW eventually

Reply:
yeah, it would be from the comments

Reply:
Personally, I'd remove the comment and definitely remove the invalid element whitelist here.

Reply:
Done